src/Overlay/Modal/Modal.styled.ts

```diff
@@ -0,0 +1,82 @@
+import styled, { css, keyframes } from 'styled-components'
+import { Button } from '../../Button'
+import { titleLarge } from '../../theme'
+
+
+const fadeInAnimation = keyframes`
+  0% {
+    opacity: 0;
+    transform: scale(0);
+  }
+  100% {
+    opacity: 1;
+    transform: scale(1);
+  }
+`
+
+export const Dialog = styled.dialog`
+    background-color: var(--md-sys-color-surface-container);
+    border: none;
+    border-radius: var(--border-radius-m);
+    flex-direction: column;
+    gap: 16px;
+    padding: 0;
+    min-width: 200px;
+    min-height: 100px;
+    max-width: 85%;
+    max-height: 85%;
+
+    /* Backdrop property affects inactive area around modal */
+    &::backdrop {
+        background-color: rgba(0, 0, 0, 0.3);
+    }
+
+    /* Styles for dialogs that carry modal behavior */
+    &:modal {
+    }
+
+    &:modal[open] {
+      animation: ${fadeInAnimation} 0.25s ease-in-out forwards, ${fadeInAnimation} 0.25s ease-in-out backwards;
+      animation-fill-mode: both;
```

Comment:
The animation doesn't seem to work when closing the dialog. Don't spend too much time on this, it's not a big deal.

It would also be nice to have an animation on the backdrop so it's not so abrupt when it appears and disappears.

Reply:
Same thing as above. If it is not essential, I will let it be as it is for now. What do you think ?